pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -129,7 +129,9 @@ func (m *Manager) Init(ctx context.Context) error {
 		return err
 	}
 	// Load resource group meta info from storage.
+	m.Lock()
```

Comment:
Do it need to be picked to other branches?

Reply:
I think so